routes/router.js

```diff
@@ -1084,25 +936,12 @@ router.route('/imageCSV')
                             res.redirect("back");
                         })
                         .catch(function(err) {
-                            //err = "Please, only upload the csv file downloaded from the project."
-                            logger.error(err);
-                            return res.format({
-                                html: function() {
-                                    res.render('errorLog', {
-                                        "error": err,
-                                    });
-                                },
-                                json: function() {
-                                    res.json(err);
-                                }
-                            });
+                            return renderError(res, err);
                         });
                 });
         }
-        
-        if (fileType == ".docx") { // Each filetype has their own route. In the future, we could also do different files for each
-            //Didn't add the classifier field to the neumes here!
 
+        if (fileType == ".docx") {
```

Comment:
Does this just check file extension? If so, there should be actual verification that files are what they should be using some node library that builds on libmagic.

Reply:
I agree, but I think there's a lot of issues with the file upload section, and I think it's worth dealing with in another branch